benchmarks/TFLite/CMakeLists.txt

```diff
@@ -158,6 +158,88 @@ set(ANDROID_MALI_GPU_TRANSLATION_FLAGS
   "--iree-vulkan-target-triple=valhall-unknown-android11"
 )
 
+set(X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-cpu=cascadelake"
+  "--iree-llvm-target-triple=x86_64-unknown-linux-gnu"
+)
+
+# CPU, Dylib-Sync, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib-sync"
+)
+
+# CPU, Dylib, 1 thread, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "1-thread,full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib"
+  RUNTIME_FLAGS
+    "--task_topology_group_count=1"
+)
+
+# CPU, Dylib, 4 threads, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "4-thread,full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib"
+  RUNTIME_FLAGS
+    "--task_topology_group_count=4"
```

Comment:
Is 4 threads what we want?

`gcp:machine-type=c2-standard-16` seems like we _could_ go up to 16 (or maybe 32?), if we wanted. We could test at several values, it just depends on what we want to be tracking and what we expect x86 users to use.

Reply:
I had some preliminary tests and it starts showing noise at 8 threads. I added the 8 threads and we can see how it goes.